moja-kuhinjica-frontend/src/pages/mealReservation/index.tsx

```diff
@@ -204,6 +212,14 @@ const MealReservation = (): JSX.Element => {
         setConfirmationModalIsOpen(false)
     }
 
+    const isBookingAllowed = (): boolean => {
+        const cutoffHour = 10
+        const today = dayjs()
+        const activeDay = dayjs(activeDate.split('/').reverse().join('/'))
+        const currentHour = today.hour()
+        return !(currentHour >= cutoffHour && today.isSame(activeDay, 'day'))
```

Comment:
rewrite this so it returns true based on conditions opposite from one that are here

I guess
`return currentHour <= cutoffHour && !(today.isSame(activeDay, 'day'))`